tests/src/hipEnvVar.cpp

```diff
@@ -67,7 +67,6 @@ int main(int argc, char** argv) {
 
     if (!env.empty()) {
         setenv("HIP_VISIBLE_DEVICES", env.c_str(), 1);
-        setenv("CUDA_VISIBLE_DEVICES", env.c_str(), 1);
```

Comment:
The test needs to run on nvcc path as well. So what you probably need is something like
```c++
#ifdef __HIP_PLATFORM_HCC__
setenv("HIP_VISIBLE_DEVICES", env.c_str(), 1);
#else
setenv("CUDA_VISIBLE_DEVICES", env.c_str(), 1);
#endif
```